### PR TITLE
Fixes #4 drawer component

### LINKED ISSUE #4: Drawer Component
Similar to Quran.com

### PR DESCRIPTION
**Pull request by mmahalwy**

### CODE REVIEW

File: src/components/Drawer/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+/* global document */
+import React, { PropTypes, Component } from 'react';
+
+const styles = require('./style.scss');
```
Comment (thabti) [inline src/components/Drawer/index.js]: why not `import styles from ./style.scss`?

Comment (mmahalwy) [reply src/components/Drawer/index.js]: From what I understand, it won't trigger watch to recompile.

**Review by thabti — APPROVED** [review]

File: src/components/Drawer/index.js (hunk 1 above)
Comment (thabti) [reply src/components/Drawer/index.js]: works fine, give it a try. woks for other components.

Comment (mmahalwy) [reply src/components/Drawer/index.js]: Hmm, okay, i'll change it!

**Pull request merged**